ospf6d/ospf6_lsa.h

```diff
@@ -55,9 +55,7 @@
 	(ospf6_lstype_debug(type) & OSPF6_LSA_DEBUG_EXAMIN)
 #define IS_OSPF6_DEBUG_FLOOD_TYPE(type)                                        \
 	(ospf6_lstype_debug(type) & OSPF6_LSA_DEBUG_FLOOD)
-#define IS_OSPF6_DEBUG_AGGR						       \
-	(ospf6_lstype_debug(OSPF6_LSTYPE_AS_EXTERNAL) & OSPF6_LSA_DEBUG_AGGR)  \
-
+#define IS_OSPF6_DEBUG_AGGR (1)
```

Comment:
This doesn't look intentional, did you enable this while hacking bits?

Reply:
ah hell
let me fix that

Reply:
fixed